### ISSUE TITLE
If the baseline return -999, should we sent the whole trace.

### ISSUE THREAD

**Issue by jorianvo** [HiSPARC/hisparcdaq#24]
The current implementation `Calculate_Trace_Variables` sets the cutoff, point from the right, to either the size of the array or to -994. The offset, point from the right, is set to 4000. This seems really complicated if we just want the whole trace.

**Comment by 153957** [comment]
Yes, if the baseline can not be determined, sent the entire trace for offline analysis.

**Comment by 153957** [comment]
Unfortunately, this probably means that a station with bad data (bad PMTs, oscillations or light leaks) and probably unusable data, produces the most data because they send all of it...

**Comment by davidfokkema** [comment]
Yes, but that _might_ make it a bit easier to diagnose, because you can see the signal in the quiet parts of the trace. Lots of moderately large peaks: light leak? Lots of small peaks: PMT with discharging dynodes? Oscillations: defective cables or PMT HV unit?

**Comment by 153957** [comment]
Yes, however, we can usually determine these same problems from the HiSPARC DAQ. But I agree we should just keep the entire traces. That way they also remain available through the Data Retrieval tool (and API) for students to look at and diagnose such problems, and having examples of what 'bad' traces look like is also good..
